drop/modules/aberrant-expression-pipeline/OUTRIDER/Summary.R

```diff
@@ -136,4 +136,10 @@ cat(paste0("<a href='./", basename(file), "'>Download OUTRIDER results table</a>
 
 res[, pValue := format(pValue, scientific = T, digits = 2)]
 res[, padjust := format(padjust, scientific = T, digits = 2)]
-DT::datatable(res, caption = "OUTRIDER results", style = 'bootstrap', filter = 'top')
+
+DT::datatable(
+  head(res, 1000),
+  caption = 'Taking a look at no more than the first 1,000 OUTRIDER results.',
```

Comment:
change the caption to: OUTRIDER results (up to 1,000 rows shown)

Reply:
also in the other 2 cases

Reply:
addressed in recent push (commit aa45cad)